static/teams/bots-graphs.html

```diff
@@ -1,173 +0,0 @@
-<div class="chart" id="keen-example-chart"></div>
```

Comment:
The bot will eventually moved here I think, so do this page really need to go in this PR?

Reply:
There is a typo in the file and https://scientific-python.org/teams/bots-graphs.html seems to be broken.

If we leave `static/teams/bots-graphs.html` in then we need to either
(a) fix the typo (but I can't see how it impacts the website),
(b) add `sucess` to the list of skip words, or
(c) tell codespell to ignore  `static/teams/bots-graphs.html`.

I am happy with any of the choices.

Reply:
Also if we leave it in, we should move it out the team gallery directory. But I wasn't sure where it should go.

Reply:
I would ignore the file in this PR, and bring in @Carreau about bot convos separately.

Reply:
Done!